bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/DeconzBridgeHandler.java

```diff
@@ -87,10 +87,11 @@ public DeconzBridgeHandler(Bridge thing, WebSocketFactory webSocketFactory, Asyn
         this.http = http;
         this.gson = gson;
         String websocketID = thing.getUID().getAsString().replace(':', '-');
-        if (websocketID.length() < 4) {
-            websocketID = "openHAB-deconz-" + websocketID;
-        } else if (websocketID.length() > 20) {
-            websocketID = websocketID.substring(websocketID.length() - 20);
+        if (websocketID.length() < 4 || websocketID.length() > 20) {
+            websocketID = "deconz-" + thing.getUID().getId();
+        }
+        if (websocketID.length() > 20) {
+            websocketID = "deconz";
```

Comment:
That's not a good idea. It should be unique, but repeatable. Maybe hash the full ID and take the last 6 bytes as hex (12 characters + `deconz-` prefix makes a total of 19 characters). I doubt that we'll run into collisions with that and it's nicely reproducible.

Reply:
It is better if it is unique but not required.
But your idea consisting in hashing the UID is good.

Reply:
Done

Reply:
Wait a second...

Reply:
That is ok now